codalab/lib/metadata_util.py

```diff
@@ -82,7 +82,7 @@ def request_missing_metadata(bundle_subclass, args, initial_metadata=None):
 
     # If args.edit exists (when doing 'cl edit'), then we want to show
     # the editor.
-    if not getattr(args, 'edit', True):
+    if getattr(args, 'edit', bypass_editor):
```

Comment:
I'm not sure this logic is right: if 'edit' is in the args, this would end up returning early without showing the editor, regardless of the value of `bypass_editor`.